manim/mobject/types/vectorized_mobject.py

```diff
@@ -508,12 +515,12 @@ def set_points_as_corners(self, points):
         ])
         return self
 
-    def set_points_smoothly(self, points):
+    def set_points_smoothly(self, points: List[Vector]):
```

Comment:
Sequence here because the function we call with `points` demands a Sequence
```suggestion
    def set_points_smoothly(self, points: typing.Sequence[Vector]) -> "SVGMobject":
```